src/wpool_queue_manager.erl

```diff
@@ -48,10 +48,12 @@
 -record(state, { wpool                 :: wpool:name()
                , clients               :: queue:queue({cast|{pid(), _}, term()})
                , workers               :: gb_sets:set(atom())
+               , monitors              :: gb_trees:tree(atom(), monitored_from())
```

Comment:
Line 51 is too long:                , monitors              :: gb_trees:tree(atom(), monitored_from()).